src/Screen/Fields/Matrix.php

```diff
@@ -33,6 +33,7 @@ class Matrix extends Field
         'maxRows'           => 0,
         'keyValue'          => false,
         'fields'            => [],
+        'addRowLabel'       => null,
```

Comment:
It seems that historically we have established a default translation pattern, can you add this? For example: https://github.com/orchidsoftware/platform/blob/5fb66a26ad6f92ec2098e18e57f583626ead262d/src/Screen/Fields/Cropper.php#L63"

Reply:
Done